packages/buckets/src/types.ts

```diff
@@ -1,4 +1,4 @@
-import CID from 'cids'
+import type { CID } from 'multiformats'
```

Comment:
we only need the types here. this is safer when building as we can tree-shake the whole dep out!